static/app/views/replays/list/replaysErroneousDeadRageCards.tsx

```diff
@@ -191,37 +165,39 @@ function CardTable({
         sort={undefined}
         visibleColumns={visibleColumns}
         saveLocation
-        gridRows={'auto ' + gridRows}
+        gridRows={`auto ${repeat(' 1fr', rows)}`}
```

Comment:
Can you use CSS [`repeat()`](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat) here?

```suggestion
        gridRows={`auto repeat(${rows}, 1fr)`}
```

Reply:
ooh, i didn't try!